bittensor/dendrite.py

```diff
@@ -267,17 +324,13 @@ async def single_axon_response(target_axon) -> bt.Synapse:
                     A Synapse object with the response.
                 """
                 if is_stream:
-                    # If in streaming mode, we iterate over the streaming content
-                    # and take the last item as the final result.
-                    final_result = None
-                    async for result in self.call_stream(
+                    # If in streaming mode, return the async_generator
+                    return self.call_stream(
```

Comment:
If query multiple axons at the same time, this will return multiple async generators for us right? Would this have any issues on the bandwidth if we were to open 25-50 streams at once?

Reply:
Good question, I'll try this out on testnet